### DISCUSSION TITLE
styles or classes

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#68] [Q&A] [answered]
How can styles or classes be changed in elements using JavaScript?

**Comment by Bunny77K9** [comment, marked as answer]
JavaScript can be used to easily modify classes and styles in an element by making use of the following syntax:

Modify styles: document.getElementById(“input”).style.fontSize = “10”;
Modify class: document.getElementById(“button”).className = “classname”;

Next up on this top Web Development interview questions and answers blog, let us take a look at the advanced set of questions.
